### DISCUSSION TITLE
2009/02/20/%E5%B8%AE%E5%BF%99%E7%BB%99%E6%88%91%E7%9A%84%E4%B9%A6%E8%B5%B7%E4%B8%AA%E5%90%8D%E5%90%A7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#253] [Announcements]
# 2009/02/20/%E5%B8%AE%E5%BF%99%E7%BB%99%E6%88%91%E7%9A%84%E4%B9%A6%E8%B5%B7%E4%B8%AA%E5%90%8D%E5%90%A7

经常浏览我博客的朋友可能都知道，我最近在忙着写一本关于LabVIEW的书。书的风格和我写的博客文章差不多，只不过更加系统和全面些。现在书已经写的差不多了，只剩一个章节还没写。再有两个月交稿，接下的时间主要用来审阅。还不清楚我这本书由来的，可以参考这个链接：http://ruanqizhen.spaces.live.com/Blog/cns!1pU-rgQVTuuWM1TX8W8PfmDA!1073.entry

https://blog.qizhen.xyz/2009/02/20/%E5%B8%AE%E5%BF%99%E7%BB%99%E6%88%91%E7%9A%84%E4%B9%A6%E8%B5%B7%E4%B8%AA%E5%90%8D%E5%90%A7

**Comment by ruanqizhen** [comment]
之前博客下的留言：

NeoYu
2009年02月23日 14:44 编辑
出一章选读一下！！

回复
Jun
2009年02月23日 13:06 编辑
就叫《我和LabVIEW》

回复
杰
2009年02月23日 02:02 编辑
我和LabVIEW这个很好，不需要副标题，个人意见，呵呵。

回复
lele
2009年02月22日 13:41 编辑
希望有本能够称作 LabVIEW编程思想 的中文书面世。不知道阮侠的书能不能做到。至于"土星"一说，如果一定要用的话，我同意楼上一位的观点，用作副标题。很期待阮侠的书！！！

回复
ateng
2009年02月21日 01:15 编辑
我觉得最主要的选择标准还是书的内容，如果适合初学者，叫跟我学LABVIEW挺不错的。如果是比较深的话就叫高级编程技巧之类的，如果是初学者或者高手都能看的就叫个从入门到精通之类的，反正多试试，再调查下取哪个名字销量最好，呵呵。我也经常到您的博客逛，看过关于LABVIEW的文章，关于算法分析的很多，说实话我没接触过LABVIEW，不过以后可能还得花时间好好学学，到时候就找您的书啦，呵呵，希望您的书能拥有更多的读者，销量更好！

回复
Wu
2009年02月21日 00:57 编辑
去网上书店找找灵感，输入c#或java，得到如下清单：c# 入门经典c# 高级编程c# 2005高级编程我觉得LabVIEW可以差不多套用啊。比如 LabVIEW 8.0 入门经典 （带个版本号比较好）自己想了一个 《LabVIEW 第一步 (LabVIEW First Step)》书大卖了别忘了我啊，哈哈！

回复
Zhen
2009年02月20日 19:25 编辑
倾向倒数第二个“来自土星的编程技巧 – 和我一起学习LabVIEW” 改成这个怎么样，LabVIEW – 和我一起学习土星话哈哈哈~

回复
Benjamin
2009年02月20日 18:53 编辑
同意楼上的意见。“土星话”这种说法，比较接近网络语言风格，而接触 LabVIEW 编程的人群，应该也不会对网络陌生。

回复
Haiying
2009年02月20日 18:28 编辑
呵呵，偶是个外外行，瞎搀和一下^_^ prefer 土星这个联接，不过不喜欢最后个选项。觉得LabVIEW该放最开端，醒目直入主题，副标题带土星。LabVIEW 编程 — 土星语录 / 土星技巧 bla bla bla ~~

回复
